IdentityCore/src/cache/MSIDKeychainTokenCache.m

```diff
@@ -562,7 +562,14 @@ - (BOOL)removeItemsWithKey:(MSIDCacheKey *)key
 - (BOOL)saveWipeInfoWithContext:(id<MSIDRequestContext>)context
```

Comment:
This pull request does not update changelog.txt.

Please consider if this change would be noticeable to a partner or user and either update changelog.txt or resolve this conversation.
<!-- changelog-comment -->